src/features/farming/crops/components/RemoveCrop.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+import { Panel } from "components/ui/Panel";
+import { Button } from "components/ui/Button";
+
+import close from "assets/icons/close.png";
+import rustyShovel from "assets/tools/rusty_shovel.png";
+
+interface Props {
+  doShow: boolean;
+  onConfirm: () => void;
+  onClose: () => void;
+}
+
+export const RemoveCrop: React.FC<Props> = ({ doShow, onConfirm, onClose }) => {
+  return (
+    <Modal centered show={doShow} onHide={onClose}>
+      <Panel>
+        <img
```

Comment:
![Screen Shot 2022-07-12 at 12 59 10 pm](https://user-images.githubusercontent.com/17863697/178399658-7d4517e7-3635-4e7c-85d5-725df9435080.png)

The alignment of the header with the absolute positioned close icon is a bit too squashed. Might be worth trying to position both using flexbox.